diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index b2251bc..910b7db 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -1,17 +1,9 @@
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/TSL.qrc)
 	set(TSL_QRC ${CMAKE_CURRENT_SOURCE_DIR}/TSL.qrc)
 else()
-	add_executable(TSLDownload TSLDownload.cpp)
-	target_link_libraries(TSLDownload Qt${QT_VERSION_MAJOR}::Network)
 	set_target_properties(TSLDownload PROPERTIES AUTOMOC OFF)
 	get_target_property(qtCore_install_prefix Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
 	get_filename_component(qtCore_install_prefix ${qtCore_install_prefix} DIRECTORY)
-	add_custom_command(
-		OUTPUT TSL.qrc
-		DEPENDS TSLDownload
-		COMMAND $<TARGET_FILE:TSLDownload> "${CMAKE_CURRENT_BINARY_DIR}" ${TSL_URL} ${TSL_INCLUDE}
-		WORKING_DIRECTORY ${qtCore_install_prefix}
-	)
 	set(TSL_QRC ${CMAKE_CURRENT_BINARY_DIR}/TSL.qrc)
 endif()
 
