diff --git a/client/CMakeLists.txt b/client/CMakeLists.txt
index 24ed6de3..44377dff 100644
--- a/client/CMakeLists.txt
+++ b/client/CMakeLists.txt
@@ -292,7 +292,7 @@ else()
 	install( FILES ${CMAKE_CURRENT_BINARY_DIR}/qdigidoc4.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1 )
 	install( FILES ee.ria.qdigidoc4.desktop DESTINATION ${CMAKE_INSTALL_DATADIR}/applications )
 	install( FILES qdigidoc4.xml DESTINATION ${CMAKE_INSTALL_DATADIR}/mime/packages )
-	install( FILES ee.ria.qdigidoc4.metainfo.xml DESTINATION /usr/share/metainfo )
+	install( FILES ee.ria.qdigidoc4.metainfo.xml DESTINATION ${CMAKE_INSTALL_DATADIR}/metainfo )
 	foreach(RES 16 22 32 48 128 256 512)
 		install(
 			FILES images/digidoc_${RES}.png
