--- qdigidoc4/client/QPKCS11.cpp 2025-06-20 17:01:32.262572166 +0300
+++ qdigidoc4/client/QPKCS11-fix.cpp     2025-06-20 17:05:53.673420188 +0300
@@ -347,7 +347,7 @@
 #elif defined(Q_OS_WIN)
 		{ "opensc-pkcs11.dll", {} },
 #else
-		{ "opensc-pkcs11.so", {} },
+		{ "/app/lib/opensc-pkcs11.so", {} },
 #if defined(Q_OS_LINUX)
 		{ "/opt/latvia-eid/lib/eidlv-pkcs11.so", "3BDB960080B1FE451F830012428F536549440F900020" }, // LV-G2
 		{ "mcard-pkcs11.so", "3B9D188131FC358031C0694D54434F5373020505D3" }, // LT-G3
